src/components/playback/playbackmanager.js

```diff
@@ -2240,13 +2240,14 @@ class PlaybackManager {
                     const streamInfo = createStreamInfoFromUrlItem(item);
                     streamInfo.fullscreen = playOptions.fullscreen;
                     getPlayerData(player).isChangingStream = false;
-                    return player.play(streamInfo).then(function () {
+                    return player.play(streamInfo).then(() => {
                         loading.hide();
                         onPlaybackStartedFn();
                         onPlaybackStarted(player, playOptions, streamInfo);
-                    }, function () {
-                        // TODO: show error message
+                    }).catch((errorCode) => {
                         self.stop(player);
+                        loading.hide();
+                        showPlaybackInfoErrorMessage(self, errorCode || '');
```

Comment:
Should there be a default error message here?

Reply:
`"ErrorDefault": "There was an error processing the request. Please try again later."`?